akvo/rsr/models/partner_site.py

```diff
@@ -181,17 +181,21 @@ def partners(self):
         """All partner organisations of all projects of the Page"""
         return self.projects().all_partners()
 
-    def projects(self):
+    def all_projects(self):
         """All projects of the Page"""
         from .project import Project
         # Get all projects associated via the Page's organisation
         if self.partner_projects:
-            fk_projects = self.organisation.published_projects().public()
+            fk_projects = self.organisation.all_projects()
         else:
-            fk_projects = Project.objects.public().published()
+            fk_projects = Project.objects.all()
```

Comment:
Doesn't this result in private projects being included if self.partner_projects is false? I guess it comes down to where all_projects() is used.